src/Uno.UI/UI/Xaml/DependencyPropertyDetails.cs

```diff
@@ -75,7 +66,7 @@ internal DependencyPropertyDetails(DependencyProperty property, Type dependencyO
 		{
 			if (!HasDefaultValueSet)
 			{
-				_defaultValue = Property.GetMetadata(_dependencyObjectType).DefaultValue;
```

Comment:
This could have already been cached by `Metadata`, so no need to re-call GetMetadata